packages/web-client/app/components/card-pay/safe-chooser-dropdown/safe-option/index.hbs

```diff
@@ -9,11 +9,18 @@
       @logoTextColor={{this.data.info.textColor}}
     />
   {{/if}}
-  <div class='safe-option__name-and-type'>
-    <span class='safe-option__name'>{{this.data.info.name}}</span>
-    {{#if this.data.type}}
-      <span class='safe-option__type'>{{this.data.type}}</span>
-    {{/if}}
-  </div>
-  <div class='safe-option__address'>{{@safe.address}}</div>
-</div>
\ No newline at end of file
+  {{#if @useUrl}}
```

Comment:
I'm unsure about this but I think it could be convenient to show the card space url here maybe?

Default:
![image](https://user-images.githubusercontent.com/273660/150000485-97eb912f-9748-4d1f-b5f0-1bd92f64c4cd.png)

With url option:
![image](https://user-images.githubusercontent.com/273660/150000647-8fa00e0d-fedd-40b0-b2c9-8b12d141c8ff.png)

Reply:
agreed! Much more intelligible

Reply:
I think showing the business ID makes sense, over showing the safe address. I'm not sure if we can or should show the card space url, since users who create business accounts may not have created card spaces yet, and may not understand the concept. The identity of a merchant seems somewhat separate from a card space it owns.

Reply:
Tying a business' display name to a card space URL might confuse a user since a Card Space has its own changeable display name.

Reply:
Thanks @Aierie. This solution does bring additional confusion (ie. "are all my businesses already accessible with this URL"?)

I'm thinking about this design - just the name and id, and the url below:

![image](https://user-images.githubusercontent.com/273660/150136810-81cc9db7-278d-4f84-a0e2-f8989f663aa2.png)

Thoughts?

Reply:
I think that does look good, and I think these changes could make the withdrawal dropdown more user-friendly as well!

Reply:
agreed… I wonder if it would be good to add a faint truncated address anyway…??? But I don’t really know how to imagine actual usage 🤔

Reply:
Possibly! I'll ask in the design channel about that, but for now I think we can go with this